packages/menu/src/Menu/Menu.stories.args.js

```diff
@@ -86,4 +86,33 @@ export const argTypes = {
       options: ['singleColumn', 'twoColumns'],
     },
   },
+
+  placement: {
+    name: 'placement',
+    type: { name: 'string', required: false },
+    description: 'Popover content placement',
+    table: {
+      defaultValue: { summary: 'bottom-start' },
+    },
+    control: {
+      type: 'select',
+      options: [
+        'auto',
+        'auto-start',
+        'auto-end',
+        'top',
+        'top-start',
+        'top-end',
+        'bottom',
+        'bottom-start',
+        'bottom-end',
+        'right',
+        'right-end',
+        'right-end',
+        'left',
+        'left-end',
+        'left-end',
```

Comment:
Looks like some repeated values are in here:
```suggestion
        'right-end',
        'left',
        'left-end',
```

Reply:
right, thanks for catching that!